data/json/mapgen/public_works.json

```diff
@@ -0,0 +1,157 @@
+        " f  ______________________________           f  ",
+        " f  ___________    sssssss                   f  ",
+        " f  ___________    sssssss                   f  ",
+        " f  ___________  |--D---+----ww-ww-ww-ww--|gFf  ",
+        " f  ___________  |....|..^|..htth.........|     ",
+        " f  ___________  |....|...+..........PPP..w     ",
+        " f  ___________  |lttl|...|..........PPP..w     ",
+        " fFFF_________FFF|----|...|ccecoS........^|     ",
+        "    ___________ss|..h.|...|---------------|     ",
+        "    ___________xs|....+....llllllll|cScScS|     ",
+        "    ___________sswddd.|............|......|     ",
+        "    ___________ssw.hd.|............+......|     ",
+        "    ___________ss|l...|..htth......|..|+|+|     ",
+        "    ___________ss|-ww-|---ww---|...|.T|T|T|     ",
+        "    ___________ssssssssssssssss|...|--|-|-|     ",
+        "    ____________,_____,_____sss|.........l|     ",
+        "    ____________,_____,_____sss+...c......w     ",
+        "    ____________,_____,_____sss+...ch...hdw     ",
+        "    ____________,_____,_____sss|^..c...ddd|     ",
+        "    ________________________sss|-www--www-|     ",
+        "    ________________________sss                 ",
+        "    ________________________sss                 ",
+        "    ________________________sss                 "
+      ],
+      "terrain": {
+        " ": [ "t_grass", "t_grass", "t_grass", "t_dirt" ],
+        "$": "t_shrub",
+        "+": "t_door_c",
+        ",": "t_pavement_y",
+        "-": "t_wall",
+        ".": "t_floor",
+        "6": "t_console",
+        "=": "t_door_locked",
+        "C": "t_floor",
+        "D": "t_door_locked_alarm",
+        "F": "t_chainfence_h",
+        "G": "t_grate",
+        "O": "t_column",
+        "P": "t_floor",
+        "S": "t_floor",
+        "T": "t_floor",
+        "^": "t_floor",
+        "_": "t_pavement",
+        "c": "t_floor",
+        "d": "t_floor",
+        "e": "t_floor",
+        "f": "t_chainfence_v",
+        "g": "t_chaingate_l",
+        "h": "t_floor",
+        "l": "t_floor",
+        "o": "t_floor",
+        "r": "t_floor",
+        "s": "t_sidewalk",
+        "t": "t_floor",
+        "w": "t_window",
+        "x": "t_console_broken",
+        "|": "t_wall"
+      },
+      "furniture": {
+        "C": "f_crate_c",
+        "P": "f_pool_table",
+        "S": "f_sink",
+        "^": "f_indoor_plant",
+        "c": "f_counter",
+        "d": "f_desk",
+        "e": "f_fridge",
+        "h": "f_chair",
+        "l": "f_locker",
+        "o": "f_bookcase",
+        "r": "f_rack",
+        "t": "f_table"
+      },
+      "toilets": {
+        "T": {}
+      },
+      "items": {
+        "P": { "item": "pool_table", "chance": 80 },
+        "e": { "item": "fridge", "chance": 80 }
+      },
+      "place_items": [
+        { "item": "hardware", "x": [ 2, 2 ], "y": [ 3, 8 ], "chance": 85 },
```

Comment:
I've used `"items"` where possible without major impact (fridge 🍨 and pool table 🎱 ) . I'd have to change `"rows"` to make use of this in other item spawn locations, so I decided to skip this until all mapgen is jsonized and easily moddable.